x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/alert_stats_collector.ts

```diff
@@ -187,20 +162,23 @@ export const getAlertsStats = async (
 
     if (isIndexExists) {
       const alertsStats = await esClient.search<unknown, AlertsStats>(getAlertsStatsQuery(index));
-
       const postureTypes = ['cspm', 'kspm', 'vuln_mgmt'] as const;
 
-      return postureTypes.map((postureType) => ({
-        posture_type: postureType,
-        rules_count: alertsStats.aggregations?.aggregations[postureType].rules_count.value,
-        alerts_count: alertsStats.aggregations?.aggregations[postureType].alerts_open.doc_count,
-        alerts_open_count:
-          alertsStats.aggregations?.aggregations[postureType].alerts_open.doc_count,
-        alerts_acknowledged_count:
-          alertsStats.aggregations?.aggregations[postureType].alerts_acknowledged.doc_count,
-        alerts_closed_count:
-          alertsStats.aggregations?.aggregations[postureType].alerts_closed.doc_count,
-      })) as CloudSecurityAlertsStats[];
+      return postureTypes
+        .filter(
+          (postureType) =>
+            alertsStats?.aggregations?.[postureType]?.doc_count &&
```

Comment:
let's move ` alertsStats?.aggregations?.[postureType]?` to a variable , wdyt?

Reply:
good one, done!